project/src/components/DetailsNavigation/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import cn from 'classnames';
+import { NAVIGATION_BUTTONS } from './constants';
+import { ucFirst } from './helper';
+import { FilmDetailsContent } from '../FilmDetailsContent';
+import {Film} from '../../types/film';
+
+type DetailsNavigationProps = {
+  film: Film;
+}
+
+export const DetailsNavigation = ({ film }:DetailsNavigationProps) => {
+  const [currentButton, setCurrentButton] = useState<string>(NAVIGATION_BUTTONS.DETAILS);
```

Comment:
```suggestion
  const [currentButton, setCurrentButton] = useState(NAVIGATION_BUTTONS.DETAILS);
```

Does ts by itself understand that it's string?

Reply:
done